src/mocker/rtm.js

```diff
@@ -85,6 +85,7 @@ rtm.stopServer = function (token) {
   return new Promise((resolve, reject) => {
     const wss = wssServers.get(token)
     if (!wss) {
+      server.close()
```

Comment:
server.close is async, we should resolveinside a callback to server.close. The callback will be invoked when all connections are closed. According to the docs:

> [server.close] will be called with an Error as its only argument if the server was not open when it was closed.

so I think it's safe to always resolve regardless of errors.

Something like `return server.close(() => resolve())` should be fine